aztk/aztklib.py

```diff
@@ -0,0 +1,37 @@
+from aztk.clusterlib import ClusterLib
+from aztk.joblib import JobLib
+import aztk.azure_api as azure_api
+import aztk.config as config
+import aztk.error as error
+
+
+class AztkLib:
```

Comment:
Hhm I think this Lib suffix doesn't feel right. Maybe just `Aztk` or something more verbose(not sure about this though) 
@paselem @jiata what do you think?

Reply:
i would just use `Aztk` if that's possible

Reply:
So we are landing on `aztk.cluster.cluster_function()` and `aztk.job.job_function()`?